tests/test_functional.py

```diff
@@ -45,6 +45,42 @@ def test_pages_401(self, testapp, db):
         testapp.get(url_for("batch.download_batch", id=1), status=401)
 
 
+class TestErrorInterface:
```

Comment:
If we have a new class for it, it would be nice to move the tests of HTTP/401 here.